content/en/docs/guides/auth.md

```diff
@@ -112,6 +112,26 @@ passed to the factory method.
   configured for POSIX filesystems.
 {{% /alert %}}
 
+#### Using OAuth token-based authentication
+
+OAuth 2.0 Protocol is the industry-standard protocol for authorization. It enables
+websites or applications to obtain limited access to user accounts using OAuth tokens.
+
+gRPC offers a set of simple APIs to integrate OAuth 2.0 into applications, streamlining authentication.
+
+At a high level, using OAuth token-based authentication includes 3 steps:
+
+1. Get OAuth token on client side.
+   * You can obtain OAuth tokens using libraries like
+   [oauth2 package](https://pkg.go.dev/golang.org/x/oauth2#section-readme) in Go and
```

Comment:
IIUC, this library is primarily for sending an existing OAuth tokens around, refreshing them, etc and it doesn't deal with generating/obtaining OAuth tokens (except for the subpackage https://pkg.go.dev/golang.org/x/oauth2/google, which is Google-specific).

Similarly the `google-auth` link below is Google-specific.

Since this top section is generic and not Google-specific, should we just assume that the user already has an existing OAuth token? WDYT?

Reply:
I see, I think that make sense, let me rephrase this part.